tendermint/src/error.rs

```diff
@@ -40,6 +40,58 @@ pub enum Kind {
     /// Signature invalid
     #[error("bad signature")]
     SignatureInvalid,
+
+    /// invalid Vote type
+    #[error("invalid Type")]
+    InvalidMessageType,
```

Comment:
Can we somehow unify the comment, message and variant name?

Reply:
Covered in the next commit.